.github/workflows/test.yml

```
@@ -1,12 +1,86 @@
 name: Tests
-on: push
+on: [pull_request, push]
 jobs:
-  tests:
+  linting:
+    name: Linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v1
```

```suggestion
        uses: actions/setup-python@v2
```
https://github.com/actions/setup-python/releases